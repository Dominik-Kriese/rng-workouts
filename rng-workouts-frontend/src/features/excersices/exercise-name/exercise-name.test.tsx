import React from 'react';
import {shallow} from 'enzyme';
import 'jest-styled-components';
import ExerciseName from './exercise-name';

describe('Exercise Name', () => {
   it('renders the headline in collapsed mode', () => {
      const name = shallow(<ExerciseName name={'Name'}/>).dive().find('p');
      expect(name).toMatchSnapshot();
   });
});
