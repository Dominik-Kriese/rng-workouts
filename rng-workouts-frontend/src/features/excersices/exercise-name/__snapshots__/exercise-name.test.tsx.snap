// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exercise Name renders the headline in collapsed mode 1`] = `
<p
  className=""
>
  Name
</p>
`;
